src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.Android.cs

```diff
@@ -13,9 +13,9 @@ namespace Windows.UI.Xaml.Controls
 {
 	public partial class DatePicker
 	{
-		public DatePicker()
+		partial void InitializePartial()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'InitializePartial'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)